src/modules/mealsMenu.js

```diff
@@ -0,0 +1,82 @@
+import { commentBtnAction } from './modal.js';
+import { likeButtonAction, getLikesNumber } from './likes.js';
+
+const menuCategories = document.querySelector('.menuCategories');
+const menuOptions = document.querySelector('.menuOptions');
+
+// const menuListUrl = 'https://www.themealdb.com/api/json/v1/1/list.php?c=list';
```

Comment:
- [ ] To keep your code professional. Kindly remove unused commented codes from it.